### PR TITLE
L2D22CL

### PR DESCRIPTION
**Pull request by evyryan**
Two abbreviations I am unsure of

### CODE REVIEW

File: kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml [hunk 1]
```diff
@@ -57,15 +57,15 @@
             <p xml:id="kzz7yh-f64843-d1e103">
                <lb ed="#V" n="113"/>CIrca litteram voluit here silitum 
       <lb ed="#V" n="114"/>dinem dei cum
```
Comment (RobbieBuhite) [inline kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml:59]:
```suggestion
      similitu<lb ed="#V" n="114" break="no"/>dinem dei cum 
```

File: kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml [hunk 2]
```diff
@@ -57,15 +57,15 @@
             <p xml:id="kzz7yh-f64843-d1e103">
                <lb ed="#V" n="113"/>CIrca litteram voluit here silitum
```
Comment (RobbieBuhite) [inline kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml:58]:
```suggestion
               <lb ed="#V" n="113"/>CIrca litteram voluit habere  
```

File: kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml [hunk 3]
```diff
@@ -57,15 +57,15 @@
...
                <lb ed="#V" n="113"/>CIrca litteram voluit here silitum 
       <lb ed="#V" n="114"/>dinem dei cum 
-      equae<lb ed="#V" n="115" break="no"/>litate quadam.
+      equa<lb ed="#V" n="115" break="no"/>litate quadam.
    </p>
             <p xml:id="kzz7yh-f64843-d1e112">
       ¶ Contra non potest appeti: quod non 
       <lb ed="#V" n="116"/>potest cogitari. Sed perfectio dei totaliter a 
-      crea<lb ed="#V" n="117" break="no"/>tura cogitari non potest: ergo mulier non potuit appetem deo equari. 
+      crea<lb ed="#V" n="117" break="no"/>tura cogitari non potest: ergo mulier non potuit appetem deo equari<!--word-->.
```
Comment (RobbieBuhite) [inline kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml:65]:
```suggestion
      crea<lb ed="#V" n="117" break="no"/>tura cogitari non potest: ergo mulier non potuit appetere deo equari<!--word-->. 
```

Comment (RobbieBuhite) [reply kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml:65]: Also unsure of this word

File: kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml [hunk 4]
```diff
@@ -57,15 +57,15 @@
...
    </p>
             <p xml:id="kzz7yh-f64843-d1e112">
       ¶ Contra non potest appeti: quod non 
       <lb ed="#V" n="116"/>potest cogitari. Sed perfectio dei totaliter a 
-      crea<lb ed="#V" n="117" break="no"/>tura cogitari non potest: ergo mulier non potuit appetem deo equari. 
+      crea<lb ed="#V" n="117" break="no"/>tura cogitari non potest: ergo mulier non potuit appetem deo equari<!--word-->. 
    </p>
             <p xml:id="kzz7yh-f64843-d1e119">
-               <lb ed="#V" n="118"/>¶ Rmnens quod sicut potest cogitari ita appeti: cogitari potest secundum 
+               <lb ed="#V" n="118"/>¶ Rmnens<!--word--> quod sicut potest cogitari ita appeti: cogitari potest secundum
```
Comment (RobbieBuhite) [inline kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml:68]:
```suggestion
               <lb ed="#V" n="118"/>¶ Respondeo quod sicut potest cogitari ita appeti: cogitari potest secundum 
```

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

File: kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml [hunk 5]
```diff
@@ -55,17 +55,17 @@
...
-      equae<lb ed="#V" n="115" break="no"/>litate quadam.
+               <lb ed="#V" n="113"/>CIrca litteram voluit habere  
+      similitu<lb ed="#V" n="114" break="no"/>dinem dei cum 
+      equa<lb ed="#V" n="115" break="no"/>litate quadam.
    </p>
             <p xml:id="kzz7yh-f64843-d1e112">
       ¶ Contra non potest appeti: quod non 
       <lb ed="#V" n="116"/>potest cogitari. Sed perfectio dei totaliter a 
-      crea<lb ed="#V" n="117" break="no"/>tura cogitari non potest: ergo mulier non potuit appetem deo equari. 
+      crea<lb ed="#V" n="117" break="no"/>tura cogitari non potest: ergo mulier non potuit appetere deo equari<!--word-->.
```
Comment (jeffreycwitt) [inline kzz7yh-f64843/cod-ve09v2_kzz7yh-f64843.xml:65]:
```suggestion
      crea<lb ed="#V" n="117" break="no"/>tura cogitari non potest: ergo mulier non potuit appetere deo equari. 
```

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**